setup.py

```diff
@@ -71,10 +69,10 @@
     },
     install_requires=[
         'numpy',
-        'tensorflow>=1.13,<1.14',  # Must be on piwheels
+        'tensorflow-gpu==2.2.0rc', # This should be changed for 2.2.0 when it's released.
```

Comment:
Change this to tensorflow/tensorflow-gpu 2.2.0 when official release comes.
Currently using 2.2.0rc2 which provides correct support for GRU layer conversion to TFLite.